.ci/install_nemu.sh

```diff
@@ -19,21 +19,23 @@ arch=$("${cidir}"/kata-arch.sh -d)
 install_nemu() {
 	local nemu_repo=$(get_version "assets.hypervisor.nemu.url")
 	local nemu_version=$(get_version "assets.hypervisor.nemu.version")
+	PACKAGING_REPO="github.com/kata-containers/packaging"
+	NEMU_TAR="kata-nemu-static.tar.gz"
 	case "$arch" in
 	x86_64)
-		local nemu_bin="qemu-system-${arch}_virt"
-		;;
-	aarch64)
-		local nemu_bin="qemu-system-${arch}"
+		local nemu_bin="nemu-system-${arch}"
 		;;
 	*)
 		die "Unsupported architecture: $arch"
 		;;
 	esac
 
-	curl -LO "${nemu_repo}/releases/download/${nemu_version}/${nemu_bin}"
-	sudo install -o root -g root -m 0755 "${nemu_bin}" "/usr/local/bin"
-	rm -rf "${nemu_bin}"
+	mkdir -p "${GOPATH}/src"
+	go get -d "${PACKAGING_REPO}" || true
+
+	prefix="/usr/local" ${GOPATH}/src/${PACKAGING_REPO}/static-build/nemu/build-static-nemu.sh
```

Comment:
It might be worth creating something like the following in `lib.sh` so that it can be reused in `install_runtime.sh` to avoid the two hard-coded values:

```sh
# Default directory to install unpackaged version of nemu into
KATA_NEMU_DESTDIR=${KATA_NEMU_DESTDIR:-/usr/local}"
```